eventmesh-common/src/main/java/org/apache/eventmesh/common/protocol/http/common/ProtocolKey.java

```diff
@@ -24,7 +24,7 @@
     public static final String VERSION = "Version";
 
     public static class ClientInstanceKey {
-        ////////////////////////////////////协议层请求方描述///////////
+        ////////////////////////////////////Protocol layer requester description///////////
```

Comment:
It seems not to be a good practice using `///////////////////////` in code.

Reply:
what about this“//Protocol layer requester description//”?

Reply:
I think it might be better not use symbols in comments, the book 《Clean code》 suggests that we should use the simplest comments.
But this is just programming style.